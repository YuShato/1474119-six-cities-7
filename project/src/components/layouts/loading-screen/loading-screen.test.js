import React from 'react';
import {render} from '@testing-library/react';
import LoadingScreen from './loading-screen';

describe('Component: LoadingScreen', () => {
  test('should be render correctly', () => {
    const {container} = render(<LoadingScreen/>);
    expect(container).toMatchSnapshot();
  });
});
