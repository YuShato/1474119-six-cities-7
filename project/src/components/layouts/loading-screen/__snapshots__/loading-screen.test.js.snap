// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: LoadingScreen should be render correctly 1`] = `
<div>
  <div
    class="loading-screen"
    data-testid="loading"
  >
    <div
      class="cities__places-container cities__places-container--empty container"
    >
      <div
        class="loadingio-spinner-double-ring-60g79lxvoha"
        style="margin: auto;"
      >
        <div
          class="ldio-2wzm9rhizr"
        >
          <div />
          <div />
          <div>
            <div />
          </div>
          <div>
            <div />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
